storage.go

```diff
@@ -2,35 +2,154 @@ package bos
 
 import (
 	"context"
+	"github.com/baidubce/bce-sdk-go/bce"
```

Comment:
It's 3rd lib here, let's format the import.

Reply:
Got it!